.github/workflows/test.yml

```
@@ -15,7 +15,7 @@ jobs:
             - name: Use Node.js
               uses: actions/setup-node@v1
```

We should update the GH action since the old version is deprecated and will stop working soon.
```suggestion
              uses: actions/setup-node@v3
```